CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 0.32.0
+
+- Patch handling is more consistent. (#1263)
+- Command `packit generate` was removed. (#1269)
+- `make check` command is now more consistent and better documented. (#1230)
+- Passing changelog from source-git repo to dist-git was fixed. (#1265)
+- There is a new `source-git` subcommand, that groups source-git related
+  commands `init` and `update-dist-git`. (#1273)
+- Packit now explicitly requires git and rpm-build. (#1276)
```

Comment:
I am not sure who has started that, but in the previous releases, the source-git notes were prefixed with `Source-git: ` and grouped.

```suggestion
- Command `packit generate` was removed. (#1269)
- `make check` command is now more consistent and better documented. (#1230)
- Source-git: Patch handling is more consistent. (#1263)
- Source-git: Passing changelog from source-git repo to dist-git was fixed. (#1265)
- Source-git: There is a new `source-git` subcommand, that groups source-git related
  commands `init` and `update-dist-git`. (#1273)
- Packit now explicitly requires git and rpm-build. (#1276)
```